symbulate/plot.py

```diff
@@ -87,19 +87,30 @@ def setup_tile(v, bins, discrete):
         v_vect = np.vectorize(v_map.get)(v)
     return v_vect, v_lab, v_pos
 
+def adjust_tile(intensity, lab, pos, bins, axis):
```

Comment:
It doesn't seem like this function should be necessary. Why not just get `v_lab` and `v_pos` correct in the function `setup_tile`?